src/modules/vtol_att_control/tailsitter.cpp

```diff
@@ -204,7 +210,7 @@ void Tailsitter::update_transition_state()
 
 			// the intial attitude setpoint for a backtransition is a combination of the current fw pitch setpoint,
 			// the yaw setpoint and zero roll since we want wings level transition
-			_q_trans_start = Eulerf(0.0f, _fw_virtual_att_sp->pitch_body, yaw_sp);
+			_q_trans_start = Eulerf(0.0f, _mc_virtual_att_sp->pitch_body, yaw_sp);
```

Comment:
@sfuhrer This was supposed to smoothly initialize transitions when there was a previous pitch setpoint from the fixed wing controller. It might have needed a check for setpoint validity though.

Reply:
yes ofc, you're right. I've adapted it, check my last commit